.travis.yml

```diff
@@ -11,3 +11,10 @@ install:
 script:
   - make lint
   - make test
+
+notifications:
+  email:
+    recipients:
+    - Travis.APA@a-bc.net
+    on_success: always
+    on_failure: always
```

Comment:
Yes. thx for pointing out
- 1. I want it for my own fork
- 2. I follow You anyway, So it may motivate me to pull.
I was not aware (and am surprised) that anyone may commit. I thought You keep control.
I assume (wrongly, may be) that only commits to 'master' are affected.
( but I would'nt mind - the are eMails are filtered anyway